### PR TITLE
Introduce fluent bit

### PR DESCRIPTION
**Pull request by Atralupus**
Base code: https://github.com/planetarium/9c-k8s-config/blob/main/9c-main/cwagent-fluent-bit.yaml

Previously, we were sending all container logs to cwl, regardless of their log level.
However, now we are sending logs with log level DBG to S3, and logs of other levels are being sent to cwl.

### CODE REVIEW

File: common/bootstrap/templates/fluent-bit/fluent-bit-role.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+metadata:
+  name: fluent-bit-role-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: fluent-bit-role
+subjects:
+  - kind: ServiceAccount
+    name: fluent-bit
+    namespace: default
```
Comment (sky1045) [inline common/bootstrap/templates/fluent-bit/fluent-bit-role.yaml:28]: seems to be not critical

**Review by area363 — APPROVED** [review]

File: common/bootstrap/templates/fluent-bit/fluent-bit.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        serviceAccount:
+          create: true
+          annotations:
+            eks.amazonaws.com/role-arn: arn:aws:iam::{{ .Values.accountId }}:role/eks-{{ .Values.clusterName }}-fluent-bit
+          name: fluent-bit
+        env:
+        - name: AWS_REGION
+          value: "us-east-2"
+        - name: CLUSTER_NAME
+          value: "9c-internal-v2"
```
Comment (sky1045) [inline common/bootstrap/templates/fluent-bit/fluent-bit.yaml:25]: as value

**Review by sky1045 — CHANGES_REQUESTED** [review]

File: common/bootstrap/templates/fluent-bit/fluent-bit.yaml (hunk 2 above)
Comment (Atralupus) [reply common/bootstrap/templates/fluent-bit/fluent-bit.yaml:25]: 🦅

Comment (Atralupus) [reply common/bootstrap/templates/fluent-bit/fluent-bit.yaml]: done!

**Review by sky1045 — APPROVED** [review]

**Pull request merged**